02-oop-inheritance/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "02-oop-inheritance",
+  "version": "0.0.0",
+  "main": "ex9.js",
+  "dep endencies": {
+    "babel-runtime": "^6.6.1",
+    "babel-polyfill": "^6.7.2"
+  },
+  "devDependencies": {
+
+    "babel-plugin-transform-class-properties": "^6.6.0",
+    "babel-plugin-transform-runtime": "^6.6.0",
+    "babel-preset-es2015": "^6.6.0"
+  },
+  "engines": {
+    "node": "^6.2.2",
+    "npm": "^3.9.5"
+  },
+  "scripts": {
```

Comment:
Please remove the other scripts that are not related with this project. _test, start, update-webdriver_